tests/beacon/helpers.py

```diff
@@ -0,0 +1,18 @@
+from eth.beacon.config import (
+    DEFAULT_CONFIG,
+)
+from eth.beacon.types.validator_record import (
+    ValidatorRecord,
+)
+
+
+def mock_validator_record(pubkey, start_dynasty=0, config=DEFAULT_CONFIG):
```

Comment:
It seems problematic that this uses `DEFAULT_CONFIG` by default, yet there is another `config` object produced by the testing fixtures...  This seems like a good way to accidentally use two different configurations within the same test.